Build.bee.cs

```diff
@@ -0,0 +1,377 @@
+        throw new Exception($"Unsupported Sdk {sdk} for {nameof(CompilerFor)}");
+    }
+
+    static void Install_Windows(NPath buildDirectory, NPath installDirectory, string targetId)
+    {
+        var binDirectory = installDirectory.Combine("bin");
+        var libDirectory = installDirectory.Combine("lib");
+        var includeDirectory = installDirectory.Combine("include");
+
+        Backend.Current.AddAction(actionName: "install",
+            targetFiles: new NPath[] {},
+            inputs: new[] {buildDirectory},
+            executableStringFor: "(robocopy"
+                + $" {buildDirectory.ToString(SlashMode.Native)}"
+                + $" {binDirectory.ToString(SlashMode.Native)}"
+                + " /s"
+                + " tbb*.dll"
+                + " tbb*.pdb"
+                + ") ^& EXIT /B 0",
+            commandLineArguments: new string[] {},
+            targetDirectories: new NPath[] {installDirectory.Combine("bin")}
+        );
+        
+        Backend.Current.AddAction(actionName: "install",
+            targetFiles: new NPath[] {},
+            inputs: new[] {buildDirectory},
+            executableStringFor: "(robocopy"
+                + $" {buildDirectory.ToString(SlashMode.Native)}"
+                + $" {libDirectory.ToString(SlashMode.Native)}"
+                + " /s"
+                + " tbb*.lib"
+                + ") ^& EXIT /B 0",
+            commandLineArguments: new string[] {},
+            targetDirectories: new NPath[] {installDirectory.Combine("lib")}
+        );
+
+        Backend.Current.AddAction(actionName: "install",
+            targetFiles: new NPath[] {},
+            inputs: new[] {buildDirectory},
+            executableStringFor: "(robocopy"
+                + " include"
+                + $" {includeDirectory.ToString(SlashMode.Native)}"
+                + " /s"
+                + ") ^& EXIT /B 0",
+            commandLineArguments: new string[] {},
+            targetDirectories: new NPath[] {installDirectory.Combine("include")}
+        );
+
+        Backend.Current.AddAliasDependency($"lib::{targetId}", new[] {
+            binDirectory,
+            libDirectory,
+            includeDirectory
+        });
+    }
+
+    static void Install_LinuxOrMacOS(NPath buildDirectory, NPath installDirectory, string targetId)
+    {
+        var libDirectory = installDirectory.Combine("lib");
+        var includeDirectory = installDirectory.Combine("include");
+
+        Backend.Current.AddAction(actionName: "install",
+            targetFiles: new NPath[] {},
+            inputs: new[] {buildDirectory},
+            executableStringFor: "find",
+            commandLineArguments: new[] {
+                buildDirectory.ToString(SlashMode.Native),
+                "-name",
+                "libtbb*.*",
+                "-exec",
+                "cp",
+                "{}",
+                libDirectory.ToString(SlashMode.Native),
+                "\\;"
+            },
```

Comment:
Have you checked if this preserves symlinks (if there are any)?

Reply:
TBB uses linker scripts rather than symlinks (e.g. `INPUT (libtbb.so.2)`)

Reply:
I mean, do the stevedore artifacts end up with two copies named like libtbb.so and libtbb.so.2 ?

Reply:
Not copies, libtbb.so.2 is the binary:
https://artifactory.prd.it.unity3d.com/ui/repos/tree/General/stevedore-testing/tbb-linux-x64/2020.3_8112b06d506202dd3ba9a15da7db70e289feffc0685915aa5fcf0460645aae1f.7z/lib/libtbb.so.2

and libtbb.so is a linker script which works in much the same way as a symbolic link:
https://artifactory.prd.it.unity3d.com/artifactory/stevedore-testing/tbb-linux-x64/2020.3_8112b06d506202dd3ba9a15da7db70e289feffc0685915aa5fcf0460645aae1f.7z!/lib/libtbb.so